app/views/notifications/_comment.html.erb

```diff
@@ -1,5 +1,5 @@
 <% commentable = comment.commentable %>
-<% comment_path = polymorphic_path([commentable.project, commentable], anchor: dom_id(comment)) %>
+<% comment_path = comment_path(comment) %>
```

Comment:
don't we need the project or the :anchor any more?

Reply:
This is using the method from `CommentsHelper` and not the Rails one.

Reply:
This is added in https://github.com/dradis/dradis-ce/pull/365

Reply:
discussed there